.htmlhintrc

```diff
@@ -0,0 +1,61 @@
+{
+  /*
+   * STANDARD
+   * =================
+   */
+  
+  // All tagnames must be in lowercase only
+  "tagname-lowercase": true,
+  
+  // All attribute names must be in lowercase
+  "attr-lowercase": true,
+  
+  // Attribute values must be enclosed by double quotemarks
+  "attr-value-double-quotes": true,
+  
+  // Attributes cannot be duplicated on a single tag
+  "attr-no-duplication": true,
+  
+  // Don't look for doctypes in the html files
+  "doctype-first": false,
+  
+  // Tags must be paired
+  "tag-pair": true,
```

Comment:
I guess that's fair, at least until we can fix the htmlhint rule upstream to heed these nuances of the official html spec.